src/stat.cc

```diff
@@ -421,8 +423,11 @@ info_get_mallstat(int size, int number, int oldnum, void *data)
     StoreEntry *sentry = (StoreEntry *)data;
 
 // format: "%12s %15s %6s %12s\n","Alloc Size","Count","Delta","Alloc/sec"
-    if (number > 0)
-        storeAppendPrintf(sentry, "%12d %15d %6d %.1f\n", size, number, number - oldnum, xdiv((number - oldnum), xm_deltat));
+    if (number > 0) {
+        const auto delta = number - oldnum;
+        const auto avg = xdiv(delta, (xm_deltat.count()/1000000.0))
```

Comment:
When upgrading to std::chrono, please avoid manual unit conversions like this. As you know from very recent experience, manual conversions are trivial to get wrong when writing and reviewing code. Use std::chrono APIs instead. For example, if you need seconds, convert to an std::chrono type representing seconds (including fractional seconds if needed).

In this particular case, it is may be best to _store_ the delta (i.e. xm_deltat) as fractional seconds, as the official code was doing. Is there some value in storing microseconds that will need to be converted to seconds on every use?